app/ide-desktop/lib/dashboard/package.json

```diff
@@ -29,8 +29,8 @@
     "@fortawesome/fontawesome-svg-core is required as a peer dependency for @fortawesome/react-fontawesome"
   ],
   "dependencies": {
-    "@aws-amplify/auth": "^5.6.5",
-    "@aws-amplify/core": "^5.8.5",
+    "@aws-amplify/auth": "5.6.5",
```

Comment:
Locked deps to avoid future possible issues with login flow.
Currently we use private methods from within the `amplify` library and a single patch could potentially break our flow